articles/150_ros_command_line_arguments.md

```diff
@@ -255,3 +321,50 @@ This signficantly increases command line verbosity, but still avoids the need fo
 Remove the need for double dash tokens (`--`), conventionally used to signify the end of CLI options for a command, by adding the `--ros-` prefix to all ROS specific command line flags e.g. `--ros-remap`, `--ros-param`, etc.
 In exchange, it makes argument extraction slightly more difficult as all options must be known ahead of time, whereas `--ros-args`-based namespacing can achieve the same with a couple rules.
 It also increases command line verbosity.
+
+## Precedence order discussion
+
+There are some issues of the current precedence rules:
+
+* Precedence orders for remapping rules and parameter assignments are inconsistent.
+* In the past, inconsistencies were detected between `rclcpp` and `rclpy` ([example issue](https://github.com/ros2/rclcpp/issues/953)).
+* Weird interaction with launch files.
+
+Example of the last item:
+
+```python
+LaunchDescription([
+  Node(
+    package='my_pkg',
+    exec='my_exec',
+    name='my_node',
+    parameters=['/path/to/my_params.yaml', {'my_int': '3'}]
+  )
+])
+```
+
+Where `my_params.yaml` can be find in *Precedence order of command line arguments* section.
```

Comment:
@ivanpauno I'd reproduce the example parameter file here.

Reply:
I will address this after we reach a conclusion on how precedence should work.
As my current goal is to start a discussion, having a clear reference to the file isn't important.